src/java/README.md

```diff
@@ -0,0 +1,5 @@
+# OT Commissioner Java
+
+**OT Commissioner Java** binds C++ classes in [include/commissioner](../../include/commissioner) to equivalent Java classes. Instead of crafting JNI and Java classes by hand, we use [SWIG](http://www.swig.org) to generate those Java classes from a defined [interface file](./commissioner.i). This simplifies the maintaining of the Commissioner interface between C++ and Java.
```

Comment:
```suggestion
**OT Commissioner Java** binds C++ classes in [include/commissioner](../../include/commissioner) to equivalent Java classes. Instead of crafting JNI and Java classes by hand, we use [SWIG](http://www.swig.org) to generate those Java classes from a defined [interface file](./commissioner.i). This simplifies the maintenance of the Commissioner interface between C++ and Java.
```